server/src/test/java/org/elasticsearch/index/mapper/vectors/DenseVectorFieldTypeTests.java

```diff
@@ -186,5 +206,21 @@ public void testByteCreateKnnQuery() {
 
         e = expectThrows(IllegalArgumentException.class, () -> cosineField.createKnnQuery(new byte[] { 0, 0, 0 }, 10, null, null));
         assertThat(e.getMessage(), containsString("The [cosine] similarity does not support vectors with zero magnitude."));
+
+        DenseVectorFieldType fieldWith2048dims = new DenseVectorFieldType(
```

Comment:
should we have a separate test method for this?

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/95257/commits/1d4b838ca1c8494673a833fdad9285e2dec4beb2